v2/internal/testcommon/kube_per_test_context.go

```diff
@@ -143,6 +144,8 @@ func (ctx KubeGlobalContext) ForTestWithConfig(t *testing.T, cfg config.Values)
 	context := context.Background() // we could consider using context.WithTimeout(OperationTimeout()) here
 	match := NewKubeMatcher(verify, context)
 
+	format.MaxLength = 0 // Disable output truncation
```

Comment:
Yeah this has annoyed me in the past too. Happy to keep it set like this as it's better IMO